extensions/2.0/Vendor/EXT_geopose/0.0.0/schema/gltf.EXT_geopose.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "EXT_geopose glTF extension",
+  "type": "object",
+  "description": "glTF extension for specifying the position and orientation of models on the Earth.",
+  "allOf": [
+    {
+      "$ref": "glTFProperty.schema.json"
+    }
+  ],
+  "properties": {
+    "longitude": {
+      "type": "number",
+      "description": "Longitude in the WGS84 datum. The angle must be specified in degrees.",
+      "minimum": -180.0,
+      "maximum": 180.0
+    },
+    "latitude": {
+      "type": "number",
+      "description": "Latitude in the WGS84 datum. The angle must be specified in degrees.",
+      "minimum": -90.0,
+      "maximum": 90.0
+    },
+    "height": {
+      "type": "number",
+      "description": "Heights are in meters above (or below) the WGS84 ellipsoid."
+    },
+    "ypr": {
```

Comment:
glTF does not use abbreviations in property names...maybe "ypr" is common enough but this might need to be renamed.  Also, perhaps the `ypr` indirection isn't needed at all, and that `yaw`, `pitch`, and `roll` could be in the top-level object.

Reply:
I would also prefer a slightly different schema but this intentionally matches the JSON encoding defined in the Geopose spec.

https://github.com/opengeospatial/GeoPose/blob/main/standard/pdf/geopose_standard.pdf - page 42

![image](https://user-images.githubusercontent.com/915398/117381031-1a5de800-aea9-11eb-9f15-6f4458ce2ac3.png)

Also on page 10:

> GeoPose 1.0 JSON encodings are specified via JSON-Schema:2019-9 and most of the requirements are that conforming encoded data objects shall validate against the corresponding schema

Reply:
@lilleyse there is a discussion point to be add with the glTF community: if this extension should try to strictly follow GeoPose norms or glTF norms.  From a glTF ecosystem perspective, consistency throughout that ecosystem may be preferred, but it is to be discussed.  There may also be lessons learned from bringing the XMP spec to glTF.

@sanjeetsuhag could you please start a **TODO** checklist at the top of this GitHub issue.